test_runner/src/main/kotlin/ftl/args/ValidateAndroidArgs.kt

```diff
@@ -32,6 +33,17 @@ fun AndroidArgs.validate() = apply {
     checkNumUniformShards()
 }
 
+private fun AndroidArgs.assertTestTargetForShards() {
+    if (testTargetsForShard != null && testTargetsForShard.isNotEmpty()) {
```

Comment:
This should fix the compilation error

```suggestion
    if (testTargetsForShard.isNotEmpty()) {
```